webapp/components/form/AutocompleteDialog/store/useLocalState.js

```diff
@@ -24,6 +24,14 @@ export const useAutocompleteDialog = ({
   )
   const Actions = useActions({ setState, onItemSelect, onClose })
 
+  useEffect(() => {
+    setState(State.assocItems(items))
+  }, [items])
```

Comment:
this is called onMount as well. To avoid double setting you can use useOnUpdate custom hook